Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/NewMeterWizard.tsx

```diff
@@ -316,6 +316,22 @@ export default function NewMeterWizard(props: {IsEngineer: boolean}) {
         localStorage.setItem('NewMeterWizard.CurrentStep', currentStep.toString())
     }
 
+    function meterDetails() {
+        setError([]);
+        setWarning([]);
+        if (meterID > 0) {
+            // Construct the URL with the meterID
+            const url = `${homePath}index.cshtml?name=Meter&MeterID=${meterID}#meterInfo`;
```

Comment:
I don't think the `#meterInfo` works. IRC you need to set the `localStorage` if you want to redirect someone to a specific tab or supply a `tab` parameter.